examples/example-jwt-auth/README.md

```diff
@@ -0,0 +1,65 @@
+Authentication Example
+==============================================
+
+This example illustrates a simple JWT-based authentication implementation in gRPC using
+ server interceptor. It uses the JJWT library to create and verify JSON Web Tokens (JWTs).
+
+The example requires grpc-java to be pre-built. Using a release tag will download the relevant binaries
+from a maven repository. But if you need the latest SNAPSHOT binaries you will need to follow
+[COMPILING](../../COMPILING.md) to build these.
+
+The source code is [here](src/main/java/io/grpc/examples/jwtauth). Please follow the
+[steps](../README.md#-to-build-the-examples) to build the examples. The build creates scripts
+`auth-server` and `auth-client` in the `build/install/examples/bin/` directory which can be
```

Comment:
The directory for me is `build/install/example-jwt-auth/bin` after following the my modified steps.